utils/xmtpRN/sync.ts

```diff
@@ -117,7 +117,7 @@ const syncClientConversationList = async (account: string) => {
     // Load the persisted conversation list
     await fetchPersistedConversationListQuery(account);
     // Streaming conversations
-    retryWithBackoff({
+    await retryWithBackoff({
```

Comment:
Something we could do in this one is not use `await` but have a .catch for each one. Since we don't want that if one has a bug to crash the others, but still want to catch the error.

Or do a `Promise.allSettled` maybe to have them all run at same time

Reply:
Removed for now, it was just there when I was trying to track down a streaming bug